api/src/main/java/gov/va/vro/api/responses/FullHealthDataAssessmentResponse.java

```diff
@@ -66,9 +66,10 @@ public class FullHealthDataAssessmentResponse {
    * @param errorMessage error message
    */
   public FullHealthDataAssessmentResponse(
-      String veteranIcn, String diagnosticCode, String errorMessage) {
+      String veteranIcn, String diagnosticCode, String errorMessage, String status) {
     this.veteranIcn = veteranIcn;
```

Comment:
I don't think it is a good practice to design the object this way - mix the error information and valid data.

Reply:
@wlin2019 good point. @jbregstone we are not even using this object when there are errors. See GlobalExceptionHandler.  So there is no need for this new status and indeed the previous errorMessage fields in this rest response object.  errorMessage is never used and with HTTP status code values there is no need for a status in the response object. Use an intermediate object instead. We already have one in EvidenceWithSummary.  That has errorMessage as well it looks like.